docs/CUSTOM-VALUES.md

```diff
@@ -0,0 +1,38 @@
+# Custom values
+
+Fabric API modules may have certain custom values set or check for the presence of specific custom values.
+
+The use of custom values ranges from enabling/disabling features to extra metadata about fabric api modules.
+
+# Provided by Fabric API
```

Comment:
Shouldn't this technically be a `##` since `#` is for `# Custom values`? (just a small nitpick)

Reply:
In that case, the following header should be a level deeper as well.